src/gps_to_pose_conversion_node.cpp

```diff
@@ -34,6 +35,9 @@ double g_covariance_orientation_x;
 double g_covariance_orientation_y;
 double g_covariance_orientation_z;
 std::string g_frame_id;
+std::string g_tf_child_frame_id;
+
+tf::TransformBroadcaster *tf_broadcaster;
```

Comment:
Best is have this as a class member in a class so you can create the whole thing after initializing...
Anyway if you want to stick to globals for this code, what you actually want is a shared ptr since you don't want to manage memory on your own. :) (As you never delete this object, etc.)